### ISSUE TITLE
RTP search in Wine registry is inefficient

### ISSUE THREAD

**Issue by Zegeri** [EasyRPG/Player#1903]
Every call of `read_rtp_registry` in `FileFinder::InitRtpPaths` makes the engine to open `$WINEPREFIX/system.reg` and `$WINEPREFIX/user.reg`, make lowercase the entire text and create a std::map with each key. During the initialization, this is done 3 times with both `system.reg` and `user.reg`. This can become the most expensive part in the engine initialization.

![flamegraph initialization](https://user-images.githubusercontent.com/4390703/65378014-1c167000-dcb3-11e9-81c7-414059eb0637.png)

This could be fixed by caching the registry files or doing a single lookup.

**Comment by Ghabry** [comment]
Okay to solve this either need to write a custom INI parser or use a custom read function for the liblcf INI reader that prefilters the lines.
First option sounds easier to implement.

**Comment by Ghabry** [comment]
I hacked together a naive parser which skips all lines except the requested one. This will still not cache the file which means both files are read 3 times. But even this naive implementation is already twice as fast as the current ini reader. I also tested a cached INI reader, time measurement:

master: 2300ms
cached ini: 1600ms
my simple parser: 1050ms

By changing the API so the simple parser tests for all keys at once an even better speedup should be possible.
